recipes/brewer2mpl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.4.1" %}
+
+package:
+  name: brewer2mpl
```

Comment:
Given that we have repos for both `pep8` and `pycodestyle`, I would favor having a repo for both. That said, it's unlikely that `brewer2mpl` will see any future updates - the primary reason for creating this feedstock was because there are other libraries that still depend on `brewer2mpl`.